### PR TITLE
PP9-17861: Add Azure Blob Upload support

### PR DESCRIPTION
**Pull request by cguedel**

### CODE REVIEW

File: src/Picturepark.SDK.V1.AzureBlob/IngestClientExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+public static class IngestClientExtensions
+{
+    public static async Task<IReadOnlyCollection<IngestFile>> UploadFilesAsync(
+        this IIngestClient client,
+        IEnumerable<string> files,
+        CancellationToken ct = default)
+    {
+        var items = new List<IngestFile>();
+
+        var container = await client.CreateIngestContainerAsync(ct);
```
Comment (pschwarzpp) [inline src/Picturepark.SDK.V1.AzureBlob/IngestClientExtensions.cs:17]: don't we need to `.ConfigureAwait(false)` everywhere in the SDK?

File: src/Picturepark.SDK.V1.AzureBlob/Picturepark.SDK.V1.AzureBlob.csproj [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net6.0</TargetFramework>
```
Comment (pschwarzpp) [inline src/Picturepark.SDK.V1.AzureBlob/Picturepark.SDK.V1.AzureBlob.csproj:4]: could we support lower?

File: src/Picturepark.SDK.V1.Tests/Clients/IngestTests.cs [hunk 3]
```diff
@@ -0,0 +1,294 @@
...
+        // Assert
+        detail.SucceededItems.Should().ContainSingle().Which.Item.GetFileMetadata().FileName.Should().Be("file.jpg");
+    }
+
+    [Fact]
+    public async Task Should_replace_content()
+    {
+        // Arrange
+        var sourceContentId = await _fixture.GetRandomContentIdAsync("fileMetadata.fileExtension:.jpg -0030_JabLtzJl8bc", 20).ConfigureAwait(false);
+        var targetContentId = await _fixture.GetRandomContentIdAsync("fileMetadata.fileExtension:.jpg -0030_JabLtzJl8bc", 20).ConfigureAwait(false);
```
Comment (pschwarzpp) [inline src/Picturepark.SDK.V1.Tests/Clients/IngestTests.cs:154]: _could_ return the same content :D

**Review by pschwarzpp — APPROVED** [review]

**Review by phoeniks-sk — APPROVED** [review]

File: src/Picturepark.SDK.V1.AzureBlob/IngestUploadOptions.cs [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+/// <summary>
+/// Options for uploading to ingest containers.
+/// </summary>
+public class IngestUploadOptions
+{
+    /// <summary>
+    /// Number of concurrent uploads.
+    /// </summary>
+    public int ConcurrentUploads { get; set; } = 4;
```
Comment (pschwarzpp) [inline src/Picturepark.SDK.V1.AzureBlob/IngestUploadOptions.cs:13]: we might want to give some hints here that this affects the number of concurrently processed files, but that we can utilize more concurrent connections and refer to the relevant BlobUploadOptions

**Review by pschwarzpp — APPROVED** [review]

**Pull request merged**